tests/omise/RefundTest.php

```diff
@@ -75,7 +75,8 @@ public function testRetrieve() {
     $charge = OmiseCharge::retrieve(self::$_charge['id']);
     $refunds = $charge->refunds();
 
-  	$refund = $refunds->retrieve($refunds->create(array('amount' => 10000))['id']);
+    $create = $refunds->create(array('amount' => 10000));
+  	$refund = $refunds->retrieve($create['id']);
```

Comment:
There's some weird indentation in this file. Please align everything at the same level with the same indent.